src/dbally/collection.py

```diff
@@ -228,26 +227,20 @@ async def ask(self, question: str, dry_run: bool = False, return_natural_respons
 
         return result
 
-    def get_similarity_indexes(self) -> Dict[AbstractSimilarityIndex, List[Tuple[str, str, str]]]:
+    def get_similarity_indexes(self) -> Dict[AbstractSimilarityIndex, List[str]]:
         """
         List all similarity indexes from all structured views in the collection.
 
         Returns:
             Dictionary with similarity indexes as keys and values containing lists of places where they are used
-            (represented by a tupple containing view name, method name and argument name)
+            (represented by a tuple containing view name, method name and argument name)
```

Comment:
This part (about it being represented by a tuple) seems to no longer be true - from what I see, you changed it to be represented by just the name itself. I'm also not sure about the change - I think it is pretty important that one is able to locate the exact location of a given index (even for the error message). There is also the question about getting `SimilarityIndexDetector` updated to work with freeform views - it would be hard for different reasons (we should probably talk about that separately) but if we were to do it (or even use a different mechanism with similar functionality), we would definitely need the location of indexes (as the detector/cli allows user to filter indexes by those locations).

Reply:
Yeah, you are right. I think we should have more generic structure for localization info (view / method / argument doesn't always make sense)